CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Use a `Box` for `Yanked` on `File` ([#11755](https://github.com/astral-sh/uv/pull/11755))
 - Use a `SmallString` for the Yanked enum ([#11715](https://github.com/astral-sh/uv/pull/11715))
 - Use boxed slices for hash vector ([#11714](https://github.com/astral-sh/uv/pull/11714))
+- Use install concurrency for bytecode compilation too ([#11615](https://github.com/astral-sh/uv/pull/11615))
```

Comment:
Not really sure where this fits. Arguably just a "Configuration" change? This is fine though.